lib/graphql/schema/scalar.rb

```diff
@@ -19,9 +19,9 @@ def kind
 
         def specified_by_url(new_url = nil)
           if new_url
-            @specified_by_url = new_url
-          elsif defined?(@specified_by_url)
-            @specified_by_url
+            directive(GraphQL::Schema::Directive::SpecifiedBy, url: new_url)
+          elsif (directive = directives.find { |dir| dir.graphql_name == "specifiedBy" })
+            directive.arguments[:url] # rubocop:disable Development/ContextIsPassedCop
```

Comment:
`Directive.arguments` is just an attr reader from what I can see but this cop doesn't know that